documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -0,0 +1,354 @@
+Radix Sort
+##########
+
+-----------------------------------------------------------
+``radix_sort`` and ``radix_sort_by_key`` Function Templates
+-----------------------------------------------------------
+
+The ``radix_sort`` and ``radix_sort_by_key`` functions sort data using the radix sort algorithm.
+The sorting is stable, ensuring the preservation of the relative order of elements with equal keys.
+The functions implement Onesweep* [#fnote1]_ algorithm variant.
+
+A synopsis of the ``radix_sort`` and ``radix_sort_by_key`` functions is provided below:
+
+.. code:: cpp
+
+   // defined in <oneapi/dpl/experimental/kernel_templates>
+
+   namespace oneapi::dpl::experimental::kt::esimd {
+
+   template <bool IsAscending = true, std::uint8_t RadixBits = 8, typename KernelParam, typename Range>
+   sycl::event
+   radix_sort(sycl::queue q, Range&& rng, KernelParam param); // (1)
```

Comment:
I prefer the function name to be visibly distinct from its parameters. The simplest way is to add a space after the name.
```suggestion
   radix_sort (sycl::queue q, Range&& rng, KernelParam param); // (1)
```

Reply:
Done